### ISSUE TITLE
macOS - missing libraries

### ISSUE THREAD

**Issue by RyanMacG** [purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL#22]
Getting the below with the latest version after installing qt5
```
Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @rpath/discord_game_sdk.dylib
  Referenced from: /Applications/tidal-rpc.app/Contents/MacOS/tidal-rpc
  Reason: image not found

Binary Images:
       0x1045b2000 -        0x1046b8ff3 +rpc.tidal (??? - ???) <82D45738-3971-30FE-96A8-AB6661159391> /Applications/tidal-rpc.app/Contents/MacOS/tidal-rpc
       0x104722000 -        0x104b69ffb +org.qt-project.QtWidgets (5.14 - 5.14.2) <35AE5FD7-A3D9-3B76-B4BF-766E06476BFA> /usr/local/opt/qt/lib/QtWidgets.framework/Versions/5/QtWidgets
       0x105be2000 -        0x105c73eff  dyld (750.5) <1F893B81-89A5-3502-8510-95B97B9F730D> /usr/lib/dyld
    0x7fff2ff16000 -     0x7fff2ff16fff  com.apple.Carbon (160 - 162) <CAA294BD-BC93-384A-8415-B254C89098FC> /System/Library/Frameworks/Carbon.framework/Versions/A/Carbon
    0x7fff31295000 -     0x7fff31714ffb  com.apple.CoreFoundation (6.9 - 1675.129) <9E632A1E-9622-33D6-BCCE-23AC16DAA6B7> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
    0x7fff3394b000 -     0x7fff33d10ff8  com.apple.Foundation (6.9 - 1675.129) <9A74FA97-7F7B-3929-B381-D9514B1E4754> /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
```

**Comment by purpl3F0x** [comment]
Can you download this [discord_game_sdk.dylib](https://github.com/purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL/blob/master/discord-game-sdk/lib/x86_64/discord_game_sdk.dylib) and put it on the same directory as the executable ?
See if that works.

**Comment by RyanMacG** [comment]
I've put it in the application directory but nothing still. The app doesn't explicitly crash anymore but it doesn't run either

**Comment by purpl3F0x** [comment]
Hmm, interesting I have no clue...
( I assume you don't expect a window to open, it's just an icon in the taskbar)

**Comment by RyanMacG** [comment]
Yeah I've not seeing anything in the taskbar or anything running in activity monitor

**Comment by purpl3F0x** [comment]
Yeah I pretty much have no idea. My osx vm is broken so I can't compile and see my self what's going on...

**Comment by PaulBGD** [comment]
I'll test on my Mac soon, been really busy lately. My bad for not testing, I guess I assumed too much that compiling was enough.

**Comment by purpl3F0x** [comment]
What if you place the lib on `./tidal-rp.app/Contents/libs/`

**Comment by puchacamilo** [comment]
> What if you place the lib on ./tidal-rp.app/Contents/libs/ ?

Hey there, I tried that and the error still happend :(

```
Dyld Error Message:
  Library not loaded: @rpath/discord_game_sdk.dylib
  Referenced from: /private/var/folders/*/tidal-rpc.app/Contents/MacOS/tidal-rpc
  Reason: image not found

Binary Images:
       0x10b6ef000 -        0x10b7f5ff3 +rpc.tidal (??? - ???) <82D45738-3971-30FE-96A8-AB6661159391> /var/folders/*/tidal-rpc.app/Contents/MacOS/tidal-rpc
       0x10b85a000 -        0x10bca5ff3 +org.qt-project.QtWidgets (5.15 - 5.15.0) <CADDCF3F-5A20-317B-9B5F-0FD958B44042> /usr/local/opt/qt/lib/QtWidgets.framework/Versions/5/QtWidgets
       0x1164f0000 -        0x11655a70f  dyld (655.1.1) <91A01B2E-622F-3FBC-8D67-AC6D5D1C0023> /usr/lib/dyld
    0x7fff3836a000 -     0x7fff3836afff  com.apple.Carbon (158 - 158) <BE5FB24D-510F-3AE6-A455-79BFDEB99AEC> /System/Library/Frameworks/Carbon.framework/Versions/A/Carbon
    0x7fff390ef000 -     0x7fff39535ff7  com.apple.CoreFoundation (6.9 - 1575.300) <FF0FED21-542E-3301-B016-B497C978C156> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
    0x7fff3b391000 -     0x7fff3b73effb  com.apple.Foundation (6.9 - 1575.300) <CA3584DA-A8AF-3931-9B53-3B1EA240575C> /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
```

**Comment by purpl3F0x** [comment]
I have no idea how to fix that. Seems library linking on osx is a bit special. Don't even have a working mac VM.

**Comment by purpl3F0x** [comment]
probably fixed on 1.3. Feel Free to open topic again
